app/src/main/java/net/gsantner/markor/model/Document.java

```diff
@@ -52,7 +53,7 @@
     private final String _fileExtension;
     private int _format = TextFormat.FORMAT_UNKNOWN;
     private String _title = "";
-    private long _modTime = 0;  // Modtime as of when the file was last loaded / written
+    private long _lastSaveTime = -1;
```

Comment:
I suggest to keep 0: `(NEW-OLD) => NEW - (-1) => (NEW+1)`. Not that it will matter much, but with 0 highest the result can get is NEW (-> `==` / `<=` / `>=` are true)